src/java/com/palantir/cassandra/cvim/CrossVpcIpMappingSynVerbHandler.java

```diff
@@ -70,7 +71,7 @@ public void doVerb(MessageIn<CrossVpcIpMappingSyn> message, int id) throws Unkno
         if (!proposedTargetName.equals(getBroadcastHostname()))
         {
             logger.warn("Received a cross VPC Syn intended for another host. Ignoring. Intended: {} Actual: {}",
-                        proposedTargetName, DatabaseDescriptor.getBroadcastAddress());
+                        proposedTargetName, broadcastAddress);
```

Comment:
instead of storing `broadcastAddress`, why not use `FBUtilities.getBroadcastAddress()` here too? avoids any potential races

Reply:
broadcast address is set in cassandra.yaml + should be install config, so no race concern. If we only called it in a single place in this class to begin with (though I left the call in #getBroadcastHostname b/c we use it for unit tests), we may not have even had this issue

Reply:
in general though even if the value could change, it could still be advantageous to lock in the value at the start of a method so it doesn't swap under the hood at a later point within a single execution (though again, the function we're using for mocking unit tests means that's not really the case here anyway)